thrift-gen/jaeger/ttypes.go

```diff
@@ -878,6 +879,7 @@ type Span struct {
 	Duration      int64      `thrift:"duration,9,required" json:"duration"`
 	Tags          []*Tag     `thrift:"tags,10" json:"tags,omitempty"`
 	Logs          []*Log     `thrift:"logs,11" json:"logs,omitempty"`
+	Incomplete    *bool      `thrift:"incomplete,12" json:"incomplete,omitempty"`
```

Comment:
What is this being generated from? I don't see corresponding `jaeger.thrift` changes. 

Also, why does thrift generate a pointer to a bool?

Reply:
You are right. The Pull Request to the IDL Repo is missing. Will do that. 

Thrift generates a pointer for all attributes, that are optional. It's the same for Tags, Logs and References. Since the collector should also accept spans from clients, that don't support the incomplete span feature, it must be optional imho.

Reply:
@vprithvi 
I created the pull request for the idl repo: https://github.com/jaegertracing/jaeger-idl/pull/50

Since I have no control over the thrift generated files, I will mark this as resolved. Please let me know, if there is still anything to be changed. Thanks!